packages/jazz-tools/src/coValues/coStream.ts

```diff
@@ -540,7 +544,28 @@ export class BinaryCoStream extends CoValueBase implements CoValue {
             allowUnfinished?: boolean;
         },
     ): Promise<Blob | undefined> {
-        const stream = await this.load(id, as, []);
+        let stream = await this.load(id, as, []);
```

Comment:
**question:** in which cases `stream` can be `undefined`?

Wondering what to do if the user is passing `allowUnfinished: true` and `stream` is undefined

Reply:
When the CoValue for the stream doesn't load at all